build.sbt

```diff
@@ -18,7 +18,8 @@ lazy val commonSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
   parallelExecution := false,
   // Sonatype OSS deployment
   credentials += Credentials(Path.userHome / ".ivy2" / ".credentials"),
-  Test / publishArtifact := false
+  Test / publishArtifact := false,
+  mimaPreviousArtifacts := Set.empty // we only use MiMa for `core` for now, using enableMimaSettings
```

Comment:
there's no core module here, I think mima is misconfigured

Reply:
Yeah result of copy-pasting i fix comments after merge this line should be delete and we will have working mima for all projects